app/src/lib/parser/index.js

```diff
@@ -1,11 +1,13 @@
 import {pipe} from '$lib/pipeline'
 import {tokenize_input} from './tokenize'
 import {check_capitalization, check_for_pronouns} from './syntax'
-import {use_alternate_lookups} from './alternate_lookups'
-import {apply_checker_rules, apply_transform_rules} from '../rules/rules_processor'
-import {perform_ontology_lookups} from '$lib/lookups'
+import {perform_form_lookups, perform_ontology_lookups} from '$lib/lookups'
 import {check_pairings} from './pairings'
 import {clausify, flatten_sentences} from './clausify'
+import {rules_applier} from '../rules/rules_processor'
+import {LOOKUP_RULES} from '../rules/lookup_rules'
```

Comment:
we could collapse these into a single `rules.index.js` that exports each of these so you have fewer import lines.